pallets/parachain-staking/src/lib.rs

```diff
@@ -379,6 +412,8 @@ pub mod pallet {
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 		/// The currency type
 		type Currency: Currency<Self::AccountId> + ReservableCurrency<Self::AccountId>;
+		/// Maximum Collator Candidates, bound collator candidate pool storage item
+		type MaxCollatorCandidates: Get<u32>;
```

Comment:
```suggestion
		type DefaultMaxCollatorCandidates: Get<u32>;
```

because `set_max_collator_candidates` allows setting this by root

TODO: add path for setting lower max_collator_candidates